templates/web/base/admin/_translations.html

```diff
@@ -0,0 +1,39 @@
+[% IF languages.size > 1 %]
+<h2>Translations</h2>
```

Comment:
All strings in templates need to be wrapped in `[% loc(…) %]` so they can be translated.